### ISSUE TITLE
Cannot Send anylonger on Qt5

### ISSUE THREAD

**Issue by marceymard** [dark-eye/com.darkeye.chatGPT#23]
The widget stopped sending request when either pressing the submit button or pressing ENTER. The issue started about a month ago. I installed the latest 0.8 version of the plasmoid but the issue remains.

Debian12
Plasma version 5.27.5
Frameworks version 5.103.0
Qt 5.15.8

**Comment by dark-eye** [comment]
Yes this seem to be an issue with ChatGPT thinking we are bots (ironically).
I'm a little busy currently so i could get my self to workaround it.

(some time it can start working again if you reload the site)

Sorry once i'll get enough free time i'll work on solving it

**Comment by greatluke** [comment]
I think the widget is working again

**Comment by dark-eye** [comment]
@greatluke Thannks I'll keep this  issue open  just  to make sure  once ill have free time   if its still  working i'll close this  issue
